code_challenges/code_wars/split_strings.py

```diff
@@ -0,0 +1,18 @@
+def split_the_string(s):
+    list_of_pairs = []
+    pairs = ""
+    for letter in s:
+        pairs += letter
+        if len(pairs) == 2:
+            list_of_pairs.append(pairs)
+            pairs = ""
+    if len(s) % 2 != 0:
+        pairs = s[-1] + "_"
+        list_of_pairs.append(pairs)
```

Comment:
Food for thought.

Is this:
```
if len(s) % 2 != 0:
    pairs = s[-1] + "_"
    list_of_pairs.append(pairs)
```
Different than:
```
if pairs:
    list_of_pairs.append(pairs + "_")
```

Reply:
I only want to add "_" at the end of the len(pairs) = 1, i dont see how the second one covers that.